src/changes/changes.xml

```diff
@@ -61,6 +61,7 @@ The <action> type attribute can be add,update,fix,remove.
     <action                   type="update" dev="ggregory">checkstyle.version 8.18 -> 8.23.</action>
     <action                   type="update" dev="ggregory">junit-jupiter 5.5.0 -> 5.5.1.</action>
     <action issue="LANG-1477" type="add" dev="jochen">Added Functions.as*, and tests thereof, as suggested by Peter Verhas</action>
+    <action issue="LANG-1475" type="fix" dev="stzx">StringUtils.unwrap incorrect throw StringIndexOutOfBoundsException.</action>
```

Comment:
Normally the `dev` is a committer, and `due-to` is the PR contributor name/alias. But I will amend this manually in my working copy while merging :+1:

Reply:
Ok, thank you for your work for this.👍